roles/etcd/templates/openssl.conf.j2

```diff
@@ -34,7 +34,7 @@ DNS.{{ idx | string }} = {{ apiserver_loadbalancer_domain_name }}
 {% set idx =  groups['etcd'] | length | int + 3 %}
 {%- for etcd_alt_name in etcd_cert_alt_names -%}
 DNS.{{ idx + 1 + loop.index }} = {{ etcd_alt_name }}
-{%- endfor -%}
+{% endfor -%}
```

Comment:
I think it should be like this
```js
{% for .. in .. %}
{% endfor %}
```

Reply:
Depends what was intended here: {% endfor %}  would introduce additional newline after the text produced by this for loop.
http://jinja.pocoo.org/docs/2.10/templates/#whitespace-control

Reply:
yep, i mean expected result is:
```
{% for k in keys %}
DNS.{{ loop.index}}={{ k }}
{% endfor %}

"DNS.1=svc\nDNS.2=etcd.svc\nDNS.3=etcd.svc.gggg\n"
```

not this
```
{%- for k in keys -%}
DNS.{{ loop.index}}={{ k }}
{%- endfor %}

"DNS.1=svcDNS.2=etcd.svcDNS.3=etcd.svc.gggg\n"
```

Reply:
i'm apply `{%` patch and reset my cluster
now `openssl.conf` looks like
```
[alt_names]
DNS.1 = localhost
DNS.2 = kube01
DNS.3 = kube02
DNS.4 = kube03
DNS.8 = etcd.kube-system.svc.adex-kube
DNS.9 = etcd.kube-system.svc
DNS.10 = etcd.kube-system
DNS.11 = etcd
IP.1 = ...

# openssl x509 -in /etc/ssl/etcd/ssl/member-kube01.pem -text | grep 'Subject Alternative Name' -A2
  X509v3 Subject Alternative Name:
    DNS:localhost, DNS:kube01, DNS:kube02, DNS:kube03, DNS:etcd.kube-system.svc.adex-kube, DNS:etcd.kube-system.svc, DNS:etcd.kube-system, DNS:etcd, IP Address:...
```

Thnx for catch this! 👍

Reply:
I am happy to help

Reply:
can you please replace `{%-` to `{%` ?
`{%- -%} {% -%}` equals `{% %} {% %}`, but it confusing 😄

Reply:
Done